README.md

```diff
@@ -36,6 +33,7 @@ return {
 ```lua
 use {
   "rawnly/gist.nvim",
+  config = function() require("gist").setup() end,
```

Comment:
Do we need setup to be called with empty params? Can't we avoid it with default values?